main.go

```diff
@@ -95,6 +95,11 @@ func SetLogger(logger log.Logger) func(delegate http.Handler) http.Handler {
 	}
 }
 
+func GetLogger(ctx context.Context) basculehttp.Logger {
+	logger := logging.GetLogger(ctx)
+	return logging.Error(logger)
```

Comment:
Don't dictate a log level here.  Let code that consumes the logger, in this case `bascule`, have control over what logging levels messages are emitted as.

Also, this is the point at which you'd set up the timestamp, caller, and other logging fields by working with the `go-kit` library.